cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,20 @@
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 1 second...")
+                time.sleep(1)
```

Comment:
You have a loop here that waits for the database connection. However, there's a potential issue: if the database is not available, the loop will run indefinitely. It would be a good idea to add a maximum number of attempts or a timeout to prevent this infinite loop. Also, catching `OperationalError` is not enough to ensure that the database is ready. You should try to make a real query to the database to ensure it's ready for connections. For example, you can use `connection.ensure_connection()` method.